src/color_table.py

```diff
@@ -41,9 +42,9 @@ def __init__(self,
                  filename_raw_data: str = 'colors.txt',
                  filename_lab_out: str = 'out_lab.txt',
                  filename_rgb_out: str = 'out_rgb.txt'):
-        self.file_raw = filename_raw_data
-        self.file_lab = filename_lab_out
-        self.file_rgb = filename_rgb_out
+        self.file_raw = os.path.join(os.path.dirname(os.path.dirname(__file__)), filename_raw_data)
```

Comment:
Don't do path operations inside this class - move them to the call site

Reply:
ok